### ISSUE TITLE
ReferenceError: values is not defined

### ISSUE THREAD

**Issue by KES777** [olifolkerd/tabulator#1647]
**Describe the bug**
Can not use `searchFunc` editor [parameter](http://tabulator.info/docs/4.1/edit)

**Tabulator Info**
- Which version of Tabulator are you using? 4.1.3
- Post a copy of your construct object if possible so we can see how your table is setup
```javascript
var tabledata = [
	{id:1, country_id: 1 },
	{id:2, country_id: 2 },
	{id:3, country_id: 3 },
	{id:4, country_id: 3 },
	{id:5, country_id: 2 },
	{id:6, country_id: 2 },
	{id:7, country_id: 1 },
	{id:8, country_id: 1 },
	{id:9, country_id: 1 },
	{id:10, country_id: 1 },
	];
  
var country_list = {
  1: "Germany",
  2: "Ukraine",
  3: "Canada",
};

//Build Tabulator
var table = new Tabulator("#example-table", {
    height:"311px",
    data: tabledata,
    columns:[
        {title:"ID", field:"id", width:40 },
        {title:"Country", field:"country_id", width:130, editor:"autocomplete", editorParams:{
            values: country_list, showListOnEmpty: true, 
            searchFunc:function(term, values){ 
                "THIS CODE IS NOT REACHED";
                return [];
            }}, 
            formatter: "lookup", formatterParams: country_list, },       
    ],
});
```

https://jsfiddle.net/KES777/34mvcwLj/22/

**To Reproduce**
See [JSFiddle](https://jsfiddle.net/KES777/34mvcwLj/22/)

**Expected behavior**
Allow developer to use custom search

**Screenshots**
![__2](https://user-images.githubusercontent.com/799021/49942718-c83e5580-feee-11e8-8347-791f5d334a85.jpg)

**Desktop (please complete the following information):**
 - OS: LinuxMint 18.3
 - Browser FF
 - Version v61

**Additional context**
When I use `autocomplete` editor I want to hide internal values of `country_id` field from user at all.

But when I trigger editing I get raw values instead of theirs replacement provided via `values` property.

![__](https://user-images.githubusercontent.com/799021/49942829-105d7800-feef-11e8-88fa-e8643c1fe843.jpg)

Here instead of `3` I expect to see `Canada`

Trying to use `searchFunc` to alter default behavior case exception

![__2](https://user-images.githubusercontent.com/799021/49942718-c83e5580-feee-11e8-8347-791f5d334a85.jpg)

**Comment by olifolkerd** [comment]
This is not a bug,

You need to use the ***lookup*** [Formatter](http://tabulator.info/docs/4.1/format) if you want to show value separately from the id.

If you are having issues please dont automatically assume they are a bug, please use [StackOverflow](https://stackoverflow.com/), it should be your first port of call to ask the wider community for advice to help resolve any challenges you may be facing.

Cheers

Oli :)

**Comment by KES777** [comment]
Would it be OK if I create pull request for #870092d change?

**Comment by olifolkerd** [comment]
This should now be resolved in the 4.1.4 update
